lib/rubocop/cop/lint/interpolation_check.rb

```diff
@@ -30,6 +30,13 @@ def on_str(node)
           add_offense(node)
         end
 
+        def autocorrect(node)
+          lambda do |corrector|
+            corrector.replace(node.loc.begin, '"')
+            corrector.replace(node.loc.end, '"')
```

Comment:
i think this will fail when correcting a string like `'"#{}"'`. it may also change the output of a string like `%{\{#{}}` (where what once was a string that equaled `"{"` it's now equal to `"\{"`. These are guesses without running it though

Reply:
Sure! I updated to use `%{...}` if a offending string contains double quoted strings.